src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Globalization;
+using System.Threading;
+using Microsoft.PowerFx;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Interpreter;
+using Microsoft.PowerFx.Types;
+using Xunit;
+using static Microsoft.PowerFx.Functions.Library;
+
+namespace Microsoft.PowerFx.Tests
+{
+    public class LanguageTest : PowerFxTest
+    {
+        // Test language
+        [Fact]
+        public void GetLanguageTest()
```

Comment:
>GetLanguageTest [](http://example.com/codeflow?start=20&length=15)

We should add "default" test for what happens when we don't set it. 

var engine = new RecalcEngine();
engine.Eval("Language()")

Reply:
Add this test